tests/onboarding_test/test_onboarding_process_mqtt.py

```diff
@@ -0,0 +1,73 @@
+import pytest
+
+from agrirouter.onboarding.onboarding import OnboardingService
+from agrirouter.environments.environments import QAEnvironment
+from agrirouter.onboarding.enums import CertificateTypes, GateWays
+from agrirouter.onboarding.parameters import OnboardParameters
+from agrirouter.onboarding.response import OnboardResponse
+from tests.data import identifier
+from tests.data.applications import CommunicationUnit
+from tests.data.onboard_response_integration_service import OnboardResponseIntegrationService
+from tests.constants import cu_recipient_endpoint_id
+
+
+@pytest.fixture
```

Comment:
Is it common practice to mix fixtures (in this case they only run once and not before each test) and test cases?

Reply:
Fixtures in pytest are used to reduce the code duplicity and are passed as arguments to the Tests/Functions.